src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/DefaultLSPDocumentFactoryTest.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Linq;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
```

Comment:
We should be moving all of the corresponding test files into their own VisualStudio.ContainedLanguage test project instead of testing it in Razor here.

Reply:
Sounds good, doing it now. 

---
In reply to: [437133336](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437133336) [](ancestors = 437133336)

Reply:
See next commit

---
In reply to: [437770395](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437770395) [](ancestors = 437770395,437133336)